ped_belief[] believes
float64 cmd_vel